### DISCUSSION TITLE
npm not found in vs code how to solve this  error ?

### DISCUSSION THREAD

**Discussion by abhiraj3070** [community/community#125805] [npm] [answered]
when I run react project in vs code this time compiler showing error the react file does not exist internal and external file, how to solve this issue !?

**Comment by 1saptarshi** [comment, marked as answer]
Ensure Node.js is installed by running `node -v` and `npm -v` in your terminal. If not installed, download and install it from the [Node.js website](https://nodejs.org/). Finally, restart VS Code to ensure it recognizes the new installation.Ensure Node.js is installed by running `node -v` and `npm -v` in your terminal. If not installed, download and install it from the [Node.js website](https://nodejs.org/). Finally, restart VS Code to ensure it recognizes the new installation.

**Comment by grandmas-favorite** [comment]
Nice job @1saptarshi! That's exactly how it's done!
